torch_xla/core/xla_model.py

```diff
@@ -42,14 +50,29 @@ def get_xla_supported_devices(devkind=None, max_devices=None):
 
 
 def xrt_world_size(defval=1):
+  """
+  Returns the WORLD_SIZE environment variable, if defined, and defval
+  otherwise.
+  """
+
   return xu.getenv_as(xenv.WORLD_SIZE, int, defval=defval)
 
 
 def get_ordinal(defval=0):
+  """
+  Returns the ORDINAL environment variable, if defined, and defval
+  otherwise.
+  """
+
   return xu.getenv_as(xenv.ORDINAL, int, defval=defval)
 
 
 def is_master_ordinal():
+  """
+  Returns true if this process or thread is the 'master' process or thread,
```

Comment:
Same here ... mentioning that the `master` ordinal is ordinal 0 would make sense IMHO.